ArgoTests/JSON/JSONFileReader.swift

```diff
@@ -1,13 +1,15 @@
 import Foundation
 
 func JSONFromFile(file: String) -> AnyObject? {
-  return NSBundle(forClass: JSONFileReader.self).pathForResource(file, ofType: "json")
-    .flatMap { NSData(contentsOfFile: $0) }
+  Bundle().pathForResource("user_without_key", ofType: "json")
+  return Bundle(for: JSONFileReader.self).pathForResource(file, ofType: "json")
+    .flatMap { URL(string: "file://\($0)") }
```

Comment:
ya looks like we lost that `NSData(contentsOfFile: String)` constructor ... and if we use the URL constructor then it needs a `file://` in front